agr_literature_service/api/routers/topic_entity_tag_router.py

```diff
@@ -133,3 +136,40 @@ def show_all_reference_tags(curie_or_reference_id: str,
 def get_map_entity_curie_to_name(curie_or_reference_id: str,
                                  db: Session = db_session):
     return topic_entity_tag_crud.get_map_entity_curie_to_name(db, curie_or_reference_id)
+
+
+def revalidate_tags_process_wrapper(already_running, email: str, delete_all_first: bool, db: Session):
```

Comment:
what calls this ?  wondering where the email comes from

Reply:
That's a wrapper function that represents a 'task' that the background process can run. The email is passed by the user through the API

Reply:
Cool, thanks